test/node/features/restify.feature

```diff
@@ -53,10 +53,9 @@ Scenario: throwing non-Error error
   And the event "unhandled" is true
   And the event "severity" equals "error"
   And the event "severityReason.type" equals "unhandledErrorMiddleware"
-  And the exception "errorClass" equals "Error"
-  And the exception "message" matches "^Handled a non-error\."
+  And the exception "errorClass" equals "InvalidError"
+  And the exception "message" matches "^restify middleware received a non-error\."
   And the exception "type" equals "nodejs"
-  And the "file" of stack frame 0 equals "node_modules/@bugsnag/plugin-restify/dist/bugsnag-restify.js"
```

Comment:
The fact that this (and other similar tests) had this stackframe made it seem like the errors came from Bugsnag. The updated stacktrace mechanism means that this is no longer the top frame, but I couldn't find a way to assert that this was _not_ equal to this string.